src/content_cao.cpp

```diff
@@ -1297,16 +1297,16 @@ void GenericCAO::processMessage(const std::string &data)
 	} else if (cmd == GENERIC_CMD_UPDATE_POSITION) {
 		// Not sent by the server if this object is an attachment.
 		// We might however get here if the server notices the object being detached before the client.
-		m_position = readV3F1000(is);
-		m_velocity = readV3F1000(is);
-		m_acceleration = readV3F1000(is);
+		m_position = readV3F32(is);
```

Comment:
m_position is the absolute position on the world, isn't it?
So it should be saved as fixed-point integer instead of float.

Reply:
But if `writeV3F1000` is used at the other end, we'll end up with a failed assertion again.

Reply:
You'd need to change the other end, too.
Converting the position to a fixed-point number for sending might work a bit faster I think.

Reply:
The entire point of this PR is to send proper floating point numbers over the network. You don't gain additional accuracy by converting a float back to an integer (and multiplying it by 1000) to send it through the connection.